python/multicorn/sqlalchemyfdw.py

```diff
@@ -401,32 +410,61 @@ def can_sort(self, sortkeys):
             return []
         return sortkeys
 
-    def explain(self, quals, columns, sortkeys=None, verbose=False):
+    def can_pushdown_upperrel(self):
+        return {
+            "groupby_supported": True,
+            "agg_functions": list(_PG_AGG_FUNC_MAPPING),
+            "operators_supported": [op for op in OPERATORS if isinstance(op, str)],
+        }
+
+    def explain(self, quals, columns, sortkeys=None, aggs=None, group_clauses=None, verbose=False):
         sortkeys = sortkeys or []
-        statement = self._build_statement(quals, columns, sortkeys)
-        return [str(statement)]
+        statement = self._build_statement(quals, columns, sortkeys, aggs=aggs, group_clauses=group_clauses)
+        return ["\n" + str(statement.compile(dialect=self.engine.dialect, compile_kwargs={"literal_binds": True})) + "\n"]
```

Comment:
This looks kinda scary (https://docs.sqlalchemy.org/en/14/faq/sqlexpressions.html#rendering-bound-parameters-inline) -- a comment here mentioning why it's required would be nice.

Reply:
The sole reason for this was to assert actual values used in the remote query, though given that the query results are also being asserted it may be an overkill.

Reply:
I've left a comment expanding on the reason for this.